backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/webservice/schema/ApiTokenService.kt

```diff
@@ -0,0 +1,89 @@
+package app.ehrenamtskarte.backend.auth.webservice.schema
+
+import app.ehrenamtskarte.backend.auth.database.AdministratorEntity
+import app.ehrenamtskarte.backend.auth.database.Administrators
+import app.ehrenamtskarte.backend.auth.database.ApiTokens
+import app.ehrenamtskarte.backend.auth.database.TOKEN_LENGTH
+import app.ehrenamtskarte.backend.auth.database.repos.ApiTokensRepository
+import app.ehrenamtskarte.backend.auth.service.Authorizer
+import app.ehrenamtskarte.backend.auth.webservice.schema.types.ApiTokenMetaData
+import app.ehrenamtskarte.backend.common.webservice.GraphQLContext
+import app.ehrenamtskarte.backend.exception.service.ForbiddenException
+import at.favre.lib.crypto.bcrypt.BCrypt
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import graphql.schema.DataFetchingEnvironment
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.eq
+import org.jetbrains.exposed.sql.and
+import org.jetbrains.exposed.sql.deleteWhere
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.transactions.transaction
+import java.security.SecureRandom
+import java.time.LocalDate
+import java.util.Base64
+
+fun getByteArrayLength() = 3 * TOKEN_LENGTH / 4 // 4*(n/3) chars are needed to represent n bytes
+
+@Suppress("unused")
+class ApiTokenService {
+    @GraphQLDescription("Creates a new api token for user import endpoint")
+    fun createApiToken(expiresIn: Int, dfe: DataFetchingEnvironment): String {
+        val context = dfe.getContext<GraphQLContext>()
+        val jwtPayload = context.enforceSignedIn()
+        val admin = AdministratorEntity.findById(jwtPayload.adminId)
```

Comment:
```suggestion
        val admin = transaction { AdministratorEntity.findById(jwtPayload.adminId) }
```

Reply:
Why? transaction only makes sense if there are multiple statements inside the transaction block.